pkg/e2e/e2e.go

```diff
@@ -134,8 +134,8 @@ func beforeSuite() bool {
 		}
 		duration, err := time.ParseDuration(viper.GetString(config.Tests.ClusterHealthChecksTimeout))
 		if err != nil {
-			log.Printf("Failed parsing health check timeout, using 2 hours: %v", err)
-			duration = time.Hour * 2
+			log.Printf("Failed parsing health check timeout: %v", err)
+			return false
```

Comment:
Cool.

Latent, but it looks like the other two conditions above might be missing `return false`?

Reply:
Yup, I am dumb. I had tunnel vision while making this change.